ha/core/support_bundle/ha_support_bundle.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import sys
+import os
+import shutil
+import tarfile
+import argparse
+
+from ha.core.error import SupportBundleError
+from cortx.utils.const import CLUSTER_CONF_LOG_KEY
+from cortx.utils.conf_store import Conf, MappedConf
+
+
+class HASupportBundle:
+    """Generate support bundle for ha."""
+    _default_path = '/tmp/cortx/support_bundle/'
+    _tar_name = 'ha'
+    _tmp_src = '/tmp/cortx/ha/'
+
+    @staticmethod
+    def generate(bundle_id: str, target_path: str, cluster_conf_url: str, **filters):
+        """Generate a tar file."""
+        duration = filters.get('duration', 'P5D')
+        size_limit = filters.get('size_limit', '500MB')
+        binlogs = filters.get('binlogs', False)
+        coredumps = filters.get('coredumps', False)
+        stacktrace = filters.get('stacktrace', False)
+        # TODO process duration, size_limit, binlogs, coredumps and stacktrace
+        # Find log dirs
+        cluster_conf = MappedConf(cluster_conf_url)
+        log_base = cluster_conf.get(CLUSTER_CONF_LOG_KEY)
+        local_base = cluster_conf.get('cortx>common>storage>local')
```

Comment:
Please use _DELIM from ha/const.py file instead of hardcoded '>'

Reply:
Yes, Done